core/scss/components/brand-bar/_brand-bar--white.scss

```diff
@@ -0,0 +1,11 @@
+@charset "UTF-8";
+
+#su-brand-bar.su-brand-bar--white {
```

Comment:
Please leave a comment and remove the ID.

Reply:
see above reply